### DISCUSSION TITLE
Existing BS Roster Editor - things we DON'T like

### DISCUSSION THREAD

**Discussion by amis92** [BSData/phalanx#13] [Ideas]
Please add any and all the *bad* features of the existing BS roster editor.

**Comment by AlexMorman** [comment]
I don't like that we can't change a force type after creating it

**Comment by AlexMorman** [comment]
I don't like how there is no easy way to view wargear details without selecting them for a unit (I think this is only on mobile??)

**Comment by AlexMorman** [comment]
I don't like how I can't drag and re-arrange forces to different locations in a multi-force army

**Comment by FarseerVeraenthis** [comment]
confusing/frustrating combination of places to click/tab when selecting options

**Comment by FarseerVeraenthis** [comment]
handling of units/models with the same wargear could be cleaner

**Comment by jlueck6** [comment]
I dislike needing to open screens or dialogs for simple inputs, e.g. incrementing/decrementing a count, selecting an item from a list that could be a dropdown. (Mobile, Android)

**Comment by jlueck6** [comment]
I dislike scrolling through a category, I would like to be able to collapse them to scroll faster (mobile, android)

**Comment by jlueck6** [comment]
I don't like the context menu you get from long-pressing a Unit or Model for a few reasons (mobile, android):

-it shows at top of screen, making it a long reach for my thumb when one-handed (and I suspect many other thumb/screen combos)
-it doesn't let me multi-select with a short-press to delete/duplicate multiple things
-All features are duplicated in the (...) menu. Maybe the redundancy is fine, personally, I think the additional (...) menu just clutters things. 
  -I would replace (...) with an Eye/view button so I can edit with one touch (anywhere but Eye), view with one touch on the Eye, then have Delete, Duplicate, Comment, Fav on the context menu after a long-press (removing icons for things that are already a one-touch op)
-no menu in Roster view, long-press just leads to edit
